pkg/policy/identity.go

```diff
@@ -70,6 +70,8 @@ type Identity struct {
 	ID NumericIdentity `json:"id"`
 	// Endpoints that have this Identity where their value is the last time they were seen.
 	Labels labels.Labels `json:"labels"`
+	// SHA256 of labels.
+	LabelsSHA256 string  `json:"labelsSHA256"`
```

Comment:
Can we be sure this field is changed at the same time a new label is added / removed / modified from the Labels map above? I think this field should be in `labels.Labels` struct. It can be fixed later on.